coalib/parsing/Globbing.py

```diff
@@ -309,7 +305,7 @@ def relative_flat_glob(dirname, basename):
     """
     if os.path.exists(os.path.join(dirname, basename)):
         return [basename]
-    return[]
+    return []
```

Comment:
Rather than add bulk to your PR, this type of trivial unrelated tidy up is fodder for newcomers : https://github.com/coala/coala/issues/4228